### DISCUSSION TITLE
zsh: file exists

### DISCUSSION THREAD

**Discussion by nibazshab** [zimfw/zimfw#470] [Q&A] [answered]
zsh will not overwrite existing files

__e.g.__
now i have a file `/tese`, i want to empty it, then i type `: > /test`, but it complain `zsh: file exists: a`

how shoud i do to make it can auto overwrite existing files?

**Comment by lnicola** [comment, marked as answer]
`: >! foo` or `setopt clobber`

**Reply by nibazshab** [reply]
ok,thank u
